src/packages/@ncigdc/modern_components/ProjectCounts/ProjectCountsDataCategory.js

```diff
@@ -132,10 +153,62 @@ export default compose(
           style: { textAlign: 'right' },
         },
         {
+          key: 'case_count_meter',
+          title: (
+            <Link
+              pathname="/repository"
+              query={{
+                filters: makeFilter([
+                  {
+                    field: 'project.project_id',
+                    value: project.project_id,
+                  },
+                ]),
+                facetTab: 'files',
```

Comment:
these two tabs should be `cases`. it should show the same tabs as when you click on the numbers next to the sparkmeter